### PR TITLE
Update gateway connectivity setup

### PR DESCRIPTION
**Pull request by vadimeisenbergibm**
Remove redundant template generation step. Related to https://github.com/istio/istio.io/pull/3479.

### CODE REVIEW

File: content/docs/setup/kubernetes/multicluster/gateways/index.md [hunk 1]
```diff
@@ -52,40 +52,24 @@ on **each** Kubernetes cluster.
...
-    [Installation with Helm](/docs/setup/kubernetes/install/helm/) instructions.
-
 1. Run the following commands in **every cluster** to deploy an identical Istio control plane
    configuration in all of them.
 
     * Create a Kubernetes secret for your generated CA certificates using a command similar to the following. See [Certificate Authority (CA) certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key) for more details.
 
         {{< text bash >}}
-        $ kubectl create namespace istio-system
+        $ kubectl create namespace istio-system --dry-run -o yaml | kubectl apply -f -
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/multicluster/gateways/index.md]: @vadimeisenbergibm Can you help me understand why the `--dry-run` flag is needed?

Comment (vadimeisenbergibm) [reply content/docs/setup/kubernetes/multicluster/gateways/index.md]: This is the same issue as in https://github.com/istio/istio.io/pull/3479, to create the `istio-system` namespace if it does not exist (if we did not create it in some step before).

We should decide on https://github.com/istio/istio.io/pull/3479, and apply the same decision here.

**Comment by vadimeisenbergibm** [comment]
Fixed upstream.

**Pull request closed without merging**